ee/src/main/resources/org/jboss/as/ee/subsystem/LocalDescriptions.properties

```diff
@@ -55,7 +55,7 @@ managed-scheduled-executor-service.context-service=The name of the context servi
 managed-scheduled-executor-service.thread-factory=The name of the thread factory to be used by the scheduled executor.
 managed-scheduled-executor-service.hung-task-threshold=The runtime, in milliseconds, for tasks to be considered hung by the scheduled executor. If 0 tasks are never considered hung.
 managed-scheduled-executor-service.long-running-tasks=Flag which hints the duration of tasks executed by the scheduled executor.
-managed-scheduled-executor-service.core-threads=The minimum number of threads to be used by the scheduled executor.
+managed-scheduled-executor-service.core-threads=Provides the number of threads to keep in the pool, even if they are idle. A value of 0 means there is no limit.
```

Comment:
This needs adjusting now.

Also, can you fix the current doc page to match this? The doc source is now in this repo so it can be in this same PR:

https://github.com/wildfly/wildfly/blob/master/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EE_Concurrency_Utilities_Configuration.adoc

Reply:
@bstansberry  I've slightly modified message (to be more similar to previous one) see https://github.com/wildfly/wildfly/pull/11814/files#diff-7b776198b867307ed303cc5a4e3bd4e6L58

Message in doc page is "The mandatory core-threads provides the number of threads to keep in the executor’s pool, even if they are idle. A value of 0 means there is no limit." - this message was correct before my PR, do we really need to change anything there?

I see only one possible inaccuracy - A value of 0 means there is no limit. This part can be replaced with something like A value of 0 means undefined.
(but even if this is correct behavior, it  seems weird to "define" a value to get "undefined" limit, so I think that current message - with no limit -  is better).  What do you think?